src/ResourceManagement/Search/AdminKeysImpl.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+namespace Microsoft.Azure.Management.Search.Fluent
```

Comment:
What is the relationship between this code and the generated code for our non-fluent .NET management SDK that lives in the `vs17dev` & `psSdkJson6` branches? If this is meant to be completely separate, why not have it in a separate repo?